main.py

```diff
@@ -1,10 +1,47 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as players_file:
+        players = json.load(players_file)
+
+    for nickname_, properties in players.items():
+        if not Race.objects.all().filter(name=properties["race"]["name"]):
```

Comment:
You can also use  Model.objects.filter(some_field="data").exists() (it is the hint from README) to check if we already have the object in database

Reply:
agree, also no need to use all method before filter )
try to use get_or_create method to simplify ur code